packages/grep.rb

```diff
@@ -0,0 +1,39 @@
+
+require 'package'
+
+class Grep < Package
+  description 'A string search utility'
+  homepage 'https://www.gnu.org/software/grep/'
+  version '3.6'
+  compatibility 'all'
+  source_url 'https://ftp.gnu.org/gnu/grep/grep-3.6.tar.xz'
+  source_sha256 '667e15e8afe189e93f9f21a7cd3a7b3f776202f417330b248c2ad4f997d9373e'
+
+  binary_url ({
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/grep-3.6-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/grep-3.6-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/grep-3.6-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/grep-3.6-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+     aarch64: '87ebec000c54b032f1e2720b03a30dfb13f14dc3a2a68a4c943daae64dc5ee34',
+      armv7l: '87ebec000c54b032f1e2720b03a30dfb13f14dc3a2a68a4c943daae64dc5ee34',
+        i686: 'a6a62d2c91c346041fc4408727e387e2cc415e304d2ca7d372a97256520d6f64',
+      x86_64: '0106cbf95ada69fc0205b4af06af4e46144156934555b5bd9376e4c00d61092f',
+  })
+
+
+  depends_on 'glibc'
+  depends_on 'pcre'
+  depends_on 'texinfo' => ':build'
+
+  def self.build
+    ENV['CFLAGS'] = '-flto'
+    system "./configure #{CREW_OPTIONS}\
+    --without-included-regex"
```

Comment:
Why without regex?  That is one of main reasons I use grep. :)

Reply:
The rationale is that you don't add it if your libc has it.

Reply:
Fair enough.  I thought for sure this would be included in coreutils but apparently not.

Reply:
from configure:
```
dnl Some installers want to be informed if we do not use our regex.
dnl For example, if the host platform uses dynamic linking and the installer
dnl knows that the grep may be invoked on other hosts with buggy libraries,
dnl then the installer should configure --with-included-regex.
```

Reply:
linuxfromscratch is not using that flag, so I am dropping it.